docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -122,6 +122,17 @@ However, repeated crashes should be investigated as they may be caused by an und
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/services-and-indexes/services/analytics-service.html[Analytics Service]
 
+=== Memcached Time Jumps (CB90057)
+
+*Background*: Time jumps occur when the cluster manager process (`beam.smp`) has not been scheduled by the CPU, or not scheduled enough for a significant period of time.
```

Comment:
`beam.smp` or `memcached`?

Reply:
This is just going off this line:
>These occur when the cluster manager process (beam.smp) has not been scheduled by the CPU, or not scheduled enough for a significant period of time.

In this support ticket: [40646](https://couchbasesupport.zendesk.com/agent/tickets/40646)

Reply:
There's two different kinds of time jumps, `memcached` and `ns_server` - the LogQL you have here will find `memcached`'s, not `ns_server`'s.